functions/go/fmt/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+image: gcr.io/kpt-fn/fmt
+description: Format resources using openapi schema
+tags:
+  - mutator
```

Comment:
It mutates yaml. But I feel it's not doing "real" mutation.
Should we use tag "formatter" or something better? @frankfarzan @Shell32-Natsu Thoughts?

Reply:
In Kptfile pipeline we have only 2 types, mutators/validators. Wouldn't a user get confused if we have new tag here ?

Reply:
How about labelling this as `mutator` and `formatter`? It is in fact mutating the configs so it's correct. Tags should also be a general/flexible mechanism and we don't need have it be limited to only `validators` and `mutators`.

Reply:
Done. Added both labels.